veg2hab/criteria.py

```diff
@@ -136,14 +136,18 @@ def check_override_geometry(cls, v):
 
         return OverrideCriterium.serialize_override_geometry(v)
 
-    # Check of de override_geometry en truth_value_outside er beide zijn of beide niet zijn
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        if (self.override_geometry is None) != (self.truth_value_outside is None):
+    @root_validator
```

Comment:
Ik haal deze denk ik even weg en vervang voor een assert, deze check wordt ook in de OverrideCriteriumIO uitgevoerd.